README.md

```diff
@@ -103,6 +103,24 @@ For sample implementations, see the [.NET Core Example](https://github.com/sendg
 
 - Please see the [.nuspec file](https://github.com/sendgrid/sendgrid-csharp/tree/master/nuspec).
 
+## Setup Environment Variables
+
+### Initial Setup
+
+```bash
+cp .env_sample .env
+```
+
+### Environment Variable
+
+Update the development environment with your [SENDGRID_API_KEY](https://app.sendgrid.com/settings/api_keys), for example:
+
+```bash
+echo "export SENDGRID_API_KEY='YOUR_API_KEY'" > sendgrid.env
```

Comment:
Can you please update for importing environment variables in windows? 
https://stackoverflow.com/questions/185208/how-do-i-get-and-set-environment-variables-in-c